### PR TITLE
Do not make masks slow to show

### PR DESCRIPTION
**Pull request by kennyt**
# Before

gif

# After

gif

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/js/components/charts/Rect.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    this.transitionRect(this.props);
+  }
+
+  componentWillUpdate(nextProps) {
+    this.transitionRect(nextProps);
+  }
+
+  transitionRect(props) {
+    let {transitionDuration, transform} = props;
+
```
Comment (jfurrow) [inline src/js/components/charts/Rect.js]: #nit can we remove this newline and add it above `d3.select...`?

File: src/js/components/charts/TimeSeriesChart.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ var ChartMixin = require('../../mixins/ChartMixin');
 var ChartStripes = require('./ChartStripes');
 var InternalStorageMixin = require('../../mixins/InternalStorageMixin');
 var Maths = require('../../utils/Maths');
```
Comment (jfurrow) [inline src/js/components/charts/TimeSeriesChart.js]: This isn't your fault, but the `Maths` module isn't being used here.

**Comment by jfurrow** [comment]
:boat:

**Pull request merged**